### PR TITLE
[#21] show version in admin

### PR DESCRIPTION
**Pull request by SonnyBA**
Fixes https://github.com/maykinmedia/open-api-framework/issues/21

**Changes**

Shows the current version of the application in the admin:

image

### CODE REVIEW

File: src/objects/conf/base.py [hunk 1]
```diff
@@ -367,8 +367,25 @@
 PROJECT_NAME = "Objects"
 SITE_TITLE = "Starting point"
 ENVIRONMENT = config("ENVIRONMENT", "")
+ENVIRONMENT_SHOWN_IN_ADMIN = True
```
Comment (annashamray) [inline src/objects/conf/base.py:370]: Why is this setting needed? I don't see it being used anywhere

Comment (SonnyBA) [reply src/objects/conf/base.py:370]: It shows the `ENVIRONMENT` setting for the admin login page (and whenever messages are shown, [see base_site.htm](https://github.com/maykinmedia/objects-api/blob/master/src/objects/templates/admin/base_site.html#L39)l):

image

Comment (annashamray) [reply src/objects/conf/base.py:370]: oh, ok!

**Review by annashamray — APPROVED** [review]
Looks good, but I agree with Steven (in other PR), that it would be nice to have it in open-api-framework

File: src/objects/conf/base.py [hunk 2]
```diff
@@ -367,8 +369,25 @@
...
+        import git
+    except ImportError:
+        GIT_SHA = None
+    else:
+        repo = git.Repo(search_parent_directories=True)
+        GIT_SHA = repo.head.object.hexsha
+else:
+    GIT_SHA = None
+
+RELEASE = config("RELEASE", GIT_SHA)
```
Comment (stevenbal) [inline src/objects/conf/base.py:389]: This will be part of the OAf base settings in the future: https://github.com/maykinmedia/objects-api/pull/413

Comment (stevenbal) [reply src/objects/conf/base.py:383]: @SonnyBA can you rebase on master? This should be part of OAF now

**Review by stevenbal — CHANGES_REQUESTED** [review]
@SonnyBA can you rebase on master? The RELEASE is now determined in base.py of OAF https://github.com/maykinmedia/objects-api/pull/413

**Review by stevenbal — APPROVED** [review]

**Pull request merged**